devops/nightly.yml

```diff
@@ -40,24 +40,35 @@ jobs:
         CXXFLAGS: -stdlib=libstdc++
         BuildType: Debug
         Asan: Off
+        LLVMDownload: On
       Clang Release:
         CC: clang
         CXX: clang++
         CXXFLAGS: -stdlib=libstdc++
         Asan: Off
         BuildType: Release
+        LLVMDownload: On
       Clang Debug (ASAN):
         CC: clang
         CXX: clang++
         CXXFLAGS: -stdlib=libstdc++
         BuildType: Debug
         Asan: On
+        LLVMDownload: On
       Clang Release (ASAN):
         CC: clang
         CXX: clang++
         CXXFLAGS: -stdlib=libstdc++
         BuildType: Release
         Asan: On
+        LLVMDownload: On
```

Comment:
I would create another set, also ran at night, with all three platforms: Win,Lin,Mac on Release mode.

I often use the Nightly build to make sure a change of mine isn't breaking the build, especially infrastructure changes.

Adding the long build to it will make it really hard to test those changes.

Reply:
I can add another one, but if you change the infrastructure, you are probably going to have to run this other build too.

Reply:
I do phased testing. First I run the CI, which is quick and checks the basic. When that's passing, I run the nightly, which can take up to an hour. The build with LLVM would probably add at least one hour on top of that, which then means I can only test 2-3 times a day.

Also, because we're on the free builders, we can only use 10 of them at the same time. The nightly build already has 8 builds, if we add LLVM to three of them (Win/Lin/Mac), it's more than 10 and one build will have to wait for another to finish, and block any other testing done by other people.

Reply:
Let's leave the LLVM build for later, it's not as simple with the too-small MS-hosted builders.